src/tools/update-fixture-sources.ts

```diff
@@ -97,21 +74,14 @@ async function overridePolymer(sourceDir: string) {
     updateFixture({
       folder: 'polymer',
       repoUrl: 'https://github.com/Polymer/polymer.git',
-      packageName: '@polymer/polymer',
```

Comment:
is this removed because we have package.json's in the repos now?

Reply:
No, because this script no longer rubs modulizer at all so it doesn't need it.